ansible/roles/products/common/resources/templates/scripts/upgrade_image.j2

```diff
@@ -6,14 +6,14 @@ GREEN='\033[0;32m'
 NC='\033[0m' # No Color
 IMAGE={{ full_image_name }}
 CID=$(docker ps -a | grep $IMAGE | awk '{print $1}' | head -n 1)
-docker pull $IMAGE
+# docker pull $IMAGE
 LATEST=`docker inspect --format {%raw%}"{{.Id}}"{%endraw%} $IMAGE`
```

Comment:
Will this still hold the date of the newest image if the latest image hasn't been pulled?